exo-go/src/cmd/configure.go

```diff
@@ -0,0 +1,121 @@
+	Run: func(cmd *cobra.Command, args []string) {
+		if printHelpIfNecessary(cmd, args) {
+			return
+		}
+		fmt.Println("We are about to configure the secret store!")
+
+		secretsBucket, awsRegion := getBucketConfig()
+		err := awsHelper.CreateSecretsStore(secretsBucket, awsRegion)
+		if err != nil {
+			log.Fatalf("Cannot create secrets store: %s", err)
+		}
+	},
+}
+
+var configureReadCmd = &cobra.Command{
+	Use:   "read",
+	Short: "Reads and prints secrets from remote secrets store",
+	Long:  "Reads and prints secrets from remote secrets store",
+	Run: func(cmd *cobra.Command, args []string) {
+		if printHelpIfNecessary(cmd, args) {
+			return
+		}
+		fmt.Println("Reading secrets store...\n") //nolint vet
+
+		secretsBucket, awsRegion := getBucketConfig()
+		secrets, err := awsHelper.ReadSecrets(secretsBucket, awsRegion)
+		if err != nil {
+			log.Fatalf("Cannot read secrets: %s", err)
+		}
+		fmt.Println(secrets)
+	},
+}
+
+var configureCreateCmd = &cobra.Command{
+	Use:   "create [secrets]",
+	Short: "Creates a secret key entry in remote secrets store",
+	Long:  "Creates a secret key entry in the remote secrets store. Should be in the form 'secret_key=secret_value'",
+	Run: func(cmd *cobra.Command, args []string) {
+		if printHelpIfNecessary(cmd, args) {
+			return
+		}
+		fmt.Println("We are about to add secrets to the secret store!")
+
+		secretsBucket, awsRegion := getBucketConfig()
+		secrets := map[string]string{}
+
+		secretName := prompt.String("Secret name")
+		if secretName != "" {
+			secretValue := prompt.StringRequired("Secret value")
+			secrets[secretName] = secretValue
+		}
+
+		for secretName != "" {
+			secretName = prompt.String("Secret name (leave blank to finish prompting)")
+			if secretName != "" {
+				secretValue := prompt.StringRequired("Secret value")
+				secrets[secretName] = secretValue
+			}
+		}
+
+		secretsPretty, err := json.MarshalIndent(secrets, "", "  ")
+		if err != nil {
+			log.Fatalf("Could not marshal secrets map: %s", err)
+		}
+		fmt.Println("You are creating these secrets:\n") //nolint vet
+		fmt.Printf("%s\n\n", string(secretsPretty))
+
+		if ok := prompt.Confirm("Do you want to continue?"); ok {
+			err = awsHelper.CreateSecrets(secrets, secretsBucket, awsRegion)
+			if err != nil {
+				log.Fatalf("Cannot create secrets: %s", err)
+			}
+		} else {
+			fmt.Println("Secret creation abandoned.")
+		}
+	},
+}
+
+func getAppConfig() types.AppConfig {
+	appDir, err := os.Getwd()
+	if err != nil {
+		panic(err)
+	}
+	appConfig, err := appConfigHelpers.GetAppConfig(appDir)
+	if err != nil {
+		log.Fatalf("Cannot read application configuration: %s", err)
+	}
+	return appConfig
+}
+
+func getBucketConfig() (string, string) {
+	appConfig := getAppConfig()
+	secretsBucket := fmt.Sprintf("%s-terraform-secrets", appConfig.Name)
+	awsRegion := "us-west-2" //TODO: read from app.yml
+	return secretsBucket, awsRegion
+}
```

Comment:
Extracting getAppConfig makes sense because that happens in multiple commands even outside of `exo configure`, but getBucketConfig really on returns two strings specific to `exo configure`. Is that worth extracting a file for? It really shouldn't be public to anything outside of this command as well.